### PR TITLE
Bug 1938636: Set logLevel for policy and recovery controllers

### PR DESCRIPTION
**Pull request by damemi**

### CODE REVIEW

**Review by damemi — COMMENTED** [review]
/bugzilla refresh

File: pkg/operator/targetconfigcontroller/targetconfigcontroller.go [hunk 1]
```diff
@@ -450,33 +450,50 @@ func managePod(ctx context.Context, configMapsGetter corev1client.ConfigMapsGett
...
 	default:
-		containerArgsWithLoglevel[0] += fmt.Sprintf(" -v=%d", 2)
+		logLevel = 2
+	}
+	// containers[0] = kube-controller-manager
+	// containers[1] = cluster-policy-controller
+	// containers[2] = kube-controller-manager-cert-syncer
+	// containers[3] = kube-controller-manager-recovery-controller
+	for i := 0; i < len(required.Spec.Containers); i++ {
+		if i == 2 {
```
Comment (soltysh) [inline pkg/operator/targetconfigcontroller/targetconfigcontroller.go:472]: Can you add here also name check of the container, just to be extra careful, this is very delicate :cold_sweat:

**Comment by damemi** [comment]
Not sure why the new test is failing, it seems to be unable to find the KCM config CR, which is what we are trying to update in order to test the loglevel:
```
=== RUN   TestLogLevel
Found configuration for host https://api.ci-op-h60gfrf1-e82b8.origin-ci-int-aws.dev.rhcloud.com:6443.
    operator_test.go:344: 
        	Error Trace:	operator_test.go:344
        	Error:      	Received unexpected error:
        	            	kubecontrollermanagers.operator.openshift.io "cluster" not found
        	Test:       	TestLogLevel
--- FAIL: TestLogLevel (0.03s)
```

**Comment by soltysh** [comment]
Other than above suggestion the change itself looks good.

**Review by soltysh — APPROVED** [review]
/lgtm

**Pull request merged**